tests/codegen/global_asm.rs

```diff
@@ -10,7 +10,6 @@
 // ignore-mips64
 // ignore-msp430
 // ignore-powerpc64
-// ignore-powerpc64le
```

Comment:
There are several targets starting with `powerpc64le`:

```
powerpc64le-unknown-freebsd
powerpc64le-unknown-linux-gnu
powerpc64le-unknown-linux-musl
```

Is `ignore-powerpc64` enough to match them?

Reply:
`ignore-powerpc64le` did not match anything. Both before and after this PR, unless the full target was specified (or the end of the target, even though that was not used anywhere), the directive looked at the `cfg`s:

```
$ rustc --print=cfg --target powerpc64le-unknown-linux-gnu
debug_assertions
panic="unwind"
target_arch="powerpc64"
target_endian="little"
target_env="gnu"
target_family="unix"
target_has_atomic="16"
target_has_atomic="32"
target_has_atomic="64"
target_has_atomic="8"
target_has_atomic="ptr"
target_os="linux"
target_pointer_width="64"
target_vendor="unknown"
unix
```

So, in this case, it would match `ignore-powerpc64`, as that's what is in the `target_arch`.